src/components/organisms/StoryList.js

```diff
@@ -1,19 +1,27 @@
 import React from 'react';
 import PropType from 'prop-types';
+import './StoryList.scss';
 
-const StoryList = ({
-  stories,
-}) => {
-  const storyItems = () => (
-    stories.map((story, i) => <li key={`item-${Math.random(i)}`}>{story.content}</li>)
-  );
-
-  return (
-    <ul>
-      {storyItems()}
-    </ul>
-  );
-};
+const StoryList = ({ stories }) => (
+  <section className="story-list">
+    {
+      stories.map((story) => (
+        <article key={story.id}>
+          <header>
+            <h3>{story.title}</h3>
+            <span className="byline">
+              by&nbsp;
```

Comment:
Not a fan of non-breaking spaces. This makes it look like the name is below the word 'by'.
Consider `<span>by<span> <span>{story.author}</span>` then CSSify those spans with a margin.
Yes, it adds some markup, but not a fan of `&nbsp`.

Reply:
actually good point. I can just do this 
```js
<span>{`by ${story.author}`}</span>
```